rules/transition_support.bzl

```diff
@@ -118,7 +118,22 @@ _apple_rule_transition = transition(
     ],
 )
 
+def _xcodeproj_rule_transition_impl(settings, attr):
+    _ignore = (settings, attr)
+    return [
+        {"@build_bazel_rules_ios//rules:local_debug_options_enabled": True},
+    ]
+
+_xcodeproj_rule_transition = transition(
+    implementation = _xcodeproj_rule_transition_impl,
+    inputs = [],
+    outputs = [
+        "@build_bazel_rules_ios//rules:local_debug_options_enabled",
+    ],
+)
+
 transition_support = struct(
     apple_rule_transition = _apple_rule_transition,
     current_apple_platform = _current_apple_platform,
+    xcodeproj_rule_transition = _xcodeproj_rule_transition,
```

Comment:
maybe it's a bit more clear to name this `force_swift_local_debug_options_transition` ?

Reply:
done